explorer/src/map.js

```diff
@@ -92,6 +98,9 @@ export function initMap(center, zoom) {
 
     mp.on('mouseleave', "all", function () {
       mp.getCanvas().style.cursor = ''
+      setTimeout(() => {
+        mp.setFeatureState({source: 'zones-hover', sourceLayer : 'vector-zones', id: currentHoverAdminId}, {hover: false})
+      }, 30)
```

Comment:
I think there may be a problem with concurrent calls between this function and `hover_hierarchy` listerner.   
Shouldn't `currentHoverAdminId` be reset to null here ?